postman/Blog.postman_collection.json

```diff
@@ -100,6 +100,35 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Delete user by id",
+			"request": {
+				"method": "PATCH",
```

Comment:
Should be DELETE method

Reply:
I've fixed problems already. See commit below.